.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!--
+    Please sign your commit and our DCO.
+
+    Details on how to sign commits can be found at https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits
```

Comment:
Do we really want to ask people to GPG sign commits? This is how I understand the sentence now. Isn't that raising the bar for contributions significantly? Unfortunately Github doesn't seem to have docs for DCO signing.

The DCO app instruction page calls it _sign-off_ to differentiate from commit _signing_.

```suggestion
    Please sign-off the Developer Certificate of Origin (DCO): https://developercertificate.org/ 
    You can sign-off commits by adding the -s / --sign-off flag to your git commit.
    More details on how to sign-off commits can be found at https://github.com/apps/dco
```